esd-group-3/src/java/com/IssueInvoiceServlet.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com;
+
+import dbcon.DBConnection;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import models.Operation;
+
+/**
+ *
+ * @author Austi
+ */
+public class IssueInvoiceServlet extends HttpServlet {  
+    protected void doPost(HttpServletRequest request, HttpServletResponse response)  
+                           throws ServletException, IOException {  
+        response.setContentType("text/html");   
+        
+        // change this jsp to wherever servlet is being called from 
+        request.getRequestDispatcher("addInvoice.jsp").include(request, response);  
+          
+        try {
+            DBConnection dbcon = new DBConnection("smartcaretest", "", "");
+            Operation new_op = new Operation();
+            HttpSession loginSession = request.getSession();
+            String employee_name = (String) loginSession.getAttribute("name");
+            
+            //  dbcon, get from employee_name above, html select, select, select, select, work out charge, select slot, init paid as false?
+            // new_op.create(dbcon, employee_userid, client_userid, date, starttime, endtime, 0, 0, false);
+            
```

Comment:
Need to remove these comments?

Reply:
This file isn't finished yet as it's going to be part of another ticket too, so these comments are just here for future me/someone to do.